libraries/SRV_Channel/SRV_Channel.h

```diff
@@ -171,6 +171,9 @@ class SRV_Channel {
     // return true if function is for a multicopter motor
     static bool is_motor(SRV_Channel::Aux_servo_function_t function);
 
+    // return true if function is any throttle type
+    static bool is_throttle(SRV_Channel::Aux_servo_function_t function);
```

Comment:
const

Reply:
function can't be const, but I at least added a const to the argument for you :)